From 87546bd96f52fe9a0c8ae022722dfc439e778bbd Mon Sep 17 00:00:00 2001
From: Deepali Chourasia <deepch23@in.ibm.com>
Date: Wed, 21 Feb 2024 09:48:25 +0000
Subject: [PATCH] fix build by adding call to find_package for CUDA

---
 cmake/Dependencies.common.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cmake/Dependencies.common.cmake b/cmake/Dependencies.common.cmake
index 0c2ba6e4..13ca3e1a 100644
--- a/cmake/Dependencies.common.cmake
+++ b/cmake/Dependencies.common.cmake
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+find_package(CUDA)
+
 ##################################################################
 # OpenCV
 ##################################################################
-- 
2.40.1

